gulpfile.js

```diff
@@ -278,6 +285,40 @@ gulp.task('package:createVSIX', () =>
   }),
 );
 
+gulp.task('l10n:bundle-download', async () => {
+  const res = await got.stream('https://github.com/microsoft/vscode-loc/archive/main.zip');
+  await new Promise((resolve, reject) =>
+    res
+      .pipe(unzipper.Parse())
+      .on('entry', entry => {
+        const match = /vscode-language-pack-(.*?)\/.+ms-vscode\.js-debug.*?\.i18n\.json$/.exec(
+          entry.path,
+        );
+        if (!match) {
+          return entry.autodrain();
+        }
+
+        const buffer = new streamBuffers.WritableStreamBuffer();
+        const locale = match[1];
+        entry.pipe(buffer).on('finish', () => {
```

Comment:
You can use the `pipeline` method of streams, as in the code example I linked above, instead of manual pipes 🙂